examples/testprograms/grpc/grpc.star

```diff
@@ -0,0 +1,7 @@
+load("@grpc", "grpc1")
+
+def on_http_get():
+    # evs = grpc1.call({"host":"localhost:9980", "service":"autokitteh.sessions.v1.SessionsService", "method":"List"})
+    evs2 = grpc1.call(host="localhost:9980", service="autokitteh.events.v1.EventsService", method="List", payload={"event_type":"get"})
```

Comment:
we need to think how to support auth here, also i think we should do something like
```
svc = grpc1.with(host='...', service='...')
svc('List', {...})
```

Reply:
We agreed to start with the simplest implementation for now, therefore this is it

We can extend it later in other PRs

So unless you think this is unusable I think we should merge it and keep iterating later

Reply:
sure, create a ticket and assign to yourself though. i didn't approve mostly because of https://github.com/autokitteh/autokitteh/pull/97#discussion_r1527618409

Reply:
Opened ENG-546

Reply:
I also think this can be easily implemented in startlark, which is much more flexible and requires less code (we don't have to add anything)

@haimzlato WDYT?

```python
svc = lambda(method, payload: grpc1.call(host="localhost:9980", service="autokitteh.events.v1.EventsService", method=method, payload=payload))

svc('List', {})
```

Reply:
i think having it in the integration is more ergonomic, maybe we can add a starlark function called `with` to do it for everything.

Reply:
see https://github.com/autokitteh/autokitteh/pull/108